content/security/index.adoc

```diff
@@ -4,48 +4,60 @@ title: "Jenkins Security"
 section: security
 ---
 
+The Jenkins project takes security seriously.
+We make every possible effort to ensure users can adequately secure their automation infrastructure.
+To that end, we work with Jenkins core and plugin developers, as well as security researchers, to fix security vulnerabilities in Jenkins in a timely manner, and to improve the security of Jenkins in general.
 
-The Jenkins project takes security seriously and makes every possible effort to
-ensure users can adequately secure their automation infrastructure. Under the
-guidance of the Jenkins Security Officer, link:/2.0/[Jenkins 2.0] (and higher)
-include more secure defaults than previous releases of Jenkins.
+link:/doc/book/operating/security/[Learn more about securing Jenkins] in the Jenkins User Handbook.
 
-Additionally, the
-link:https://wiki.jenkins-ci.org/display/JENKINS/Governance+Board#GovernanceBoard-Security[Jenkins
-Security Officer], with help from members of
-link:https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+CERT+team[Jenkins CERT]
-work with security researchers, plugin and core developers to release security
-fixes and advisories for plugin and core in a timely manner.
+== Security Advisories
 
-// TODO: Once there is a good Handbook chapter it should be prominently linked
-// from here
+See our link:/security/advisories/[archive of past security advisories] for previously published security advisories.
 
+== Security Notifications
 
-== Security Advisories
+We use the +jenkinsci-advisories+ Google group to inform users about upcoming, scheduled security updates:
 
-From time to time, we issue a security advisory to report security problems in
-Jenkins. You can receive notifications for such advisories in one of the
-following ways:
+- link:https://groups.google.com/forum/#!forum/jenkinsci-advisories[+jenkinsci-advisories+ list archive]
+- link:https://feeds.feedburner.com/jenkins-security-advisories[+jenkinsci-advisories+ RSS feed]
 
-. jenkinsci-advisories@googlegroups.com (_read-only_) mailing list (link:https://groups.google.com/forum/#!forum/jenkinsci-advisories[list archive])
-. link:https://feeds.feedburner.com/jenkins-security-advisories[jenkins-security-advisories RSS feed]
+Once a security update has been released, we inform the +jenkinsci-advisories+ and link:http://oss-security.openwall.org/wiki/mailing-lists/oss-security[+oss-security+] mailing lists.
```

Comment:
I would reword this, to atleast "Whenever a security update is released," 

Or maybe something like this:
```
We also send an announcement whenever a security update is released.   These announcements are sent to the +jenkinsci-advisories+ group and the
link:http://oss-security.openwall.org/wiki/mailing-lists/oss-security[+oss-security+] mailing list.
```